src/index.html

```diff
@@ -11,7 +11,215 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="surname-input">
+          Surname:
+            <input
+              type="text"
+              name="Surname"
+              id="Surname-input"
+              required
+              autocomplete="off"
+            >
+          </label>
```

Comment:
you have different `for` and `id` attributes. please check all places
```suggestion
          <label for="surname-input">
          Surname:
            <input
              type="text"
              name="Surname"
              id="surname-input"
              required
              autocomplete="off"
            >
          </label>
```